### ISSUE TITLE
Header use

### ISSUE THREAD

**Issue by Cheesycats** [NHSDigital/data-viz-community-of-practice#7]
For easy navigation for those with screen readers the sub-headings in the below image should be H3 rather than bold font. 

![image](https://github.com/user-attachments/assets/79f88142-2d75-40a9-8db6-079917572a53)

This issue is consistent throughout the site
